ipaserver/plugins/ca.py

```diff
@@ -138,6 +138,12 @@ class ca(LDAPObject):
             ],
             'default_privileges': {'CA Administrator'},
         },
+        'System: Grant add on CAs container': {
+            'ipapermtargetfilter': {'(objectclass=nscontainer)'},
+            'ipapermbindruletype': 'permission',
+            'ipapermright': {'add'},
+            'default_privileges': {'CA Administrator'},
+        },
```

Comment:
I think we do not want to add a new permission for this; the existing 'System: Add CA' permission
should be sufficient.